launcher/Makefile

```diff
@@ -3,7 +3,7 @@ update-pip-requirements: ## Updates all Python requirements files via pip-compil
 	pip-compile --allow-unsafe --generate-hashes --output-file=dev-requirements.txt dev-requirements.in
 
 .PHONY: venv
-venv: ## Provision a Python 3 virtualenv for development (run apt-get install python3-pyqt4)
+venv: ## Provision a Python 3 virtualenv for development (run apt-get install python3-pyqt5)
```

Comment:
Yes. I confirm this is still required and the package name is correct for Debian 11.

So is installing `make`, `python3-venv` and `xvfb` on Debian 11... (not sure what to do of that, maybe we could create a target to install the -dev- system dependencies? Out of scope for this PR.)

**Edit**: for future reference, that's only `Xvfb` and `python3-PyQt5` on Fedora 36 (case sensitive!).